edgelet/iotedge/src/config/apply.rs

```diff
@@ -237,8 +237,8 @@ async fn execute_inner(
                 .await
                 .map_err(|err| format!("{:?}", err))?;
             if &hostname != new_hostname && !containers.is_empty() {
-                return Err(format!("Cannot apply config because the hostname in the config {} is different from the previous hostname {}. To update the hostname, run the following command which deletes all modules and reapplies the configuration. Or, revert the hostname change in the config.toml file.
-                    sudo iotedge system stop && sudo docker rm -f $(docker ps -aq) && sudo iotedge config apply.
+                return Err(format!("Cannot apply config because the hostname in the config {} is different from the previous hostname {}. To update the hostname, run the following command which deletes all IoT Edge modules and reapplies the configuration. Or, revert the hostname change in the config.toml file.
+                    sudo iotedge system stop && sudo docker rm -f $(docker ps -aq -f \"label=net.azure-devices.edge.owner=Microsoft.Azure.Devices.Edge.Agent\")) && sudo iotedge config apply.
```

Comment:
should the `docker ps` be with `sudo` as well?

Reply:
yes, I'd think so. Since its running in a subshell, youd need to elevate privileges there as well